paas/install_client.go

```diff
@@ -236,6 +236,20 @@ func (c *InstallClient) handleExtensions(action string, extensions []string, opt
 	return nil
 }
 
+func (c *InstallClient) listRegisteredExtensions(options *kubernetes.InstallationOptions) error {
+
+	exts, err := deployments.GetRegisteredExtensions(options)
+	if err != nil {
+		return err
+	}
+	c.ui.Note().Msg("Registered FuseML extensions:")
+	for _, extID := range exts {
+		c.ui.Normal().Msg(extID)
```

Comment:
We can discuss what should be actually printed as part of list command...

Reply:
I've changed the output format to table

Reply:
perfect :)